_pages/en_US/installing-boot9strap-(mset9-cli).txt

```diff
@@ -61,7 +61,7 @@ In this section, you will prepare the SD card data necessary for the MSET9 explo
     + Ensure that the correct console model and version is displayed
 1. Type `1`, then press Enter to ensure your console is prepared for MSET9
     + If you get an error, check the [troubleshooting](troubleshooting#installing-boot9strap-mset9), then try again
-1. If you see the message "Everything appears to be functional!", MSET9 is ready to be injected. Press Enter to close the MSET9 script
+1. If you see the message "Everything appears to be functional!", MSET9 is ready to be injected, however do not inject yet as you will do that in the next section. Press Enter to close the MSET9 script
```

Comment:
I feel like we can just reword this to 'if you see the message everything appears to be functional, continue to the next section'
That way people won't keep interpreting this as 'inject now' and it'll be a bit less word salad-y

Reply:
"If you see the message "Everything appears to be functional!", press Enter to close the MSET9 script and continue to the next step"

how's that?

Reply:
Lgtm